doc(release) Release notes gorgone 25.10.1 (#4844)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-25.10/releases/centreon-os.mdx

## Centreon Gorgone

### 25.10.1

Release date: `November 17, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Packaging] Fixed a issue that caused an upgrade failure due to centreon-perl-libs split into centreon-perl-libs and centreon-perl-libs-common.

Reviewer: ```suggestion
- [Packaging] Upgrade no longer fails because of centreon-perl-libs being split into centreon-perl-libs and centreon-perl-libs-common.
```
